pyproject.toml

```diff
@@ -26,7 +26,6 @@ dependencies = [
     "fastapi[all]<0.99",
     "uvicorn",
     "requests",
```

Comment:
There appear to be some references left to this removed package that are failing the tests.